plugins/flytekit-inference/flytekitplugins/inference/sidecar_template.py

```diff
@@ -138,6 +140,16 @@ def __init__(
                         V1VolumeMount(name="shared-data", mount_path="/shared"),
                         V1VolumeMount(name="tmp", mount_path="/tmp"),
                     ],
+                    resources=V1ResourceRequirements(
+                        requests={
+                            "cpu": 1,
```

Comment:
I can see the need for setting the CPU as well. From my testing, downloads are usually faster if you give it more than one CPU.

Reply:
i made it configurable.